pyzo/codeeditor/base.py

```diff
@@ -225,6 +225,16 @@ def __init__(self,*args, **kwds):
         # Apply a good default style
         self.setStyle(S)
     
+     # see https://bugreports.qt.io/browse/QTBUG-57552?focusedCommentId=469402&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-469402
+     # and https://code.qt.io/cgit/qt/qtbase.git/tree/src/gui/text/qtextdocument.cpp#n1183
+     # and https://doc.qt.io/qt-5/qchar.html
+    _plainTextTrans = str.maketrans({"\u2029":"\n", "\u2028":"\n", "\ufdd0":"\n", "\ufdd1":"\n"})
```

Comment:
Is this related to our frequent transaltion of `\u2029` to `\n`, e.g. [here](https://github.com/pyzo/pyzo/blob/master/pyzo/core/editorTabs.py#L448)?

Reply:
I cannot answer since I do not know the reason for the translation in ``editorTabs.py``; it may be, if QPlainTextEdit uses ``\u2029`` to represent paragraph and this corresponds to a line-break in the code. I just mimicked the behaviour of toPlainText except for nbsp. I since learned that Qt 5.9 introcuced a toRawText method that does not do any replacement.